.github/workflows/cd.yml

```
@@ -214,7 +214,9 @@ jobs:
     - name: Upload DMG
       if: env.UPLOAD_TO == 'ftp' && matrix.platform == 'macOS'
       run: |
+        mv ${PWD}/kiwix-${VERSION}.dmg ${PWD}/kiwix-desktop-macos_${VERSION}.dmg
         python .github/upload_file.py --src ${PWD}/kiwix-${VERSION}.dmg --dest ci@master.download.kiwix.org:30022/data/download/${UPLOAD_FOLDER} --ssh-key ${SSH_KEY}
+        mv ${PWD}/kiwix-desktop-macos_${VERSION}.dmg ${PWD}/kiwix-${VERSION}.dmg
```

hmm I am not sure about this one...
we rename it, but the upload script still gets the `kiwix-${VERSION}.dmg` file as the `--src` input.
Maybe it would be enough to do:
```suggestion
        python .github/upload_file.py --src ${PWD}/kiwix-${VERSION}.dmg --dest ci@master.download.kiwix.org:30022/data/download/${UPLOAD_FOLDER}/kiwix-desktop-macos_${VERSION}.dmg --ssh-key ${SSH_KEY}
```

---

My bad ; fixed it. No we can't pass full destination as the script expects a folder.

---

do we need the 2nd `mv` command after the upload ?

---

This filename is a requirement for this very step only. Changing expectations from this step onward seems unjustified. We dont **need** it because it's kind of the last step but it's cleaner IMO.